tfx/experimental/distributed_inference/graphdef_experiments/subgraph_partitioning/graph_partition_test.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for Graph Partitioning."""
+
+import os
+import tempfile
+import tensorflow as tf
+
+from create_complex_graph import save_examples_as_graphdefs
+from graph_partition import _RemoteOpLayers
+from graph_partition import get_graph_name_to_graph_def, partition_all_graphs
+
+
+class RelationTest(tf.test.TestCase):
```

Comment:
You renamed the class to `_RemoteOpLayers`. Change the test name as well.

Reply:
Changed.